target/portfolio-1/index.html

```diff
@@ -2,37 +2,107 @@
 <html lang="en" dir="ltr">
     <head>
         <meta charset="utf-8">
```

Comment:
Why is this file in portfolio-1? There is a duplicate of this index.html in webapp

Reply:
I have no idea why it's in portfolio 1. It seems that whenever I create/change the name of the file, it goes into the target folder. Idk if that happens automatically because I don't think I did that. I can look into it, though!